[0m12:21:18.042252 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002C461BD4170>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002C45F8004D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002C45F801490>]}


============================== 12:21:18.047258 | bc3ae1e9-a779-4495-bc48-d9ec5d703605 ==============================
[0m12:21:18.047258 [info ] [MainThread]: Running with dbt=1.9.2
[0m12:21:18.047258 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\konst\\.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init dbt_project', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m12:21:18.065860 [debug] [MainThread]: Starter project path: C:\Users\konst\AppData\Roaming\Python\Python312\site-packages\dbt\include\starter_project
[0m12:21:18.080920 [info ] [MainThread]: 
Your new dbt project "dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:21:18.081914 [info ] [MainThread]: Setting up your profile.
[0m12:23:12.579950 [info ] [MainThread]: Profile dbt_project written to C:\Users\konst\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m12:23:12.581966 [debug] [MainThread]: Command `dbt init` succeeded at 12:23:12.581966 after 114.68 seconds
[0m12:23:12.581966 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002C461BD4170>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002C4624AFA10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002C4624AFA40>]}
[0m12:23:12.581966 [debug] [MainThread]: Flushing usage events
[0m12:23:13.038641 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m12:23:28.988946 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000230939E4380>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000230943BD010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000230943BD640>]}


============================== 12:23:28.993026 | 95b9c9f9-e03b-4adf-a3ef-859c0f2ec252 ==============================
[0m12:23:28.993026 [info ] [MainThread]: Running with dbt=1.9.2
[0m12:23:28.994035 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\konst\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m12:23:29.005054 [info ] [MainThread]: dbt version: 1.9.2
[0m12:23:29.005946 [info ] [MainThread]: python version: 3.12.6
[0m12:23:29.005946 [info ] [MainThread]: python path: C:\Python312\python.exe
[0m12:23:29.005946 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m12:23:29.050945 [info ] [MainThread]: Using profiles dir at C:\Users\konst\.dbt
[0m12:23:29.051945 [info ] [MainThread]: Using profiles.yml file at C:\Users\konst\.dbt\profiles.yml
[0m12:23:29.053014 [info ] [MainThread]: Using dbt_project.yml file at D:\Repositories\Git_GitHub\courses_certificates_journals\Analytics_Engineering\dbt_project.yml
[0m12:23:29.053014 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m12:23:29.054079 [debug] [MainThread]: Command `dbt debug` failed at 12:23:29.054079 after 0.21 seconds
[0m12:23:29.054079 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000230942BC170>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002309475E8D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023094B4D6A0>]}
[0m12:23:29.054079 [debug] [MainThread]: Flushing usage events
[0m12:23:29.475160 [debug] [MainThread]: An error was encountered while trying to flush usage events
